src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -945,6 +960,86 @@ internal bool CalculateUseSiteDiagnostic(ref DiagnosticInfo result)
             return false;
         }
 
+#nullable enable
+        protected bool CalculateUnmanagedCallersOnlyDiagnostic(ref DiagnosticInfo result)
```

Comment:
>CalculateUnmanagedCallersOnlyDiagnostic [](start = 23, length = 39)

I couldn't find where this method is used.

Reply:
It was used at one point and I moved it mostly into the binder, but forgot to remove this one. Thanks.

---
In reply to: [481378914](https://github.com/dotnet/roslyn/pull/47311#discussion_r481378914) [](ancestors = 481378914)